.vscode/launch.json

```diff
@@ -17,6 +17,16 @@
       "request": "launch",
       "type": "node-terminal",
       "cwd": "${workspaceFolder}/frontend/"
+    },
+    {
+      "type": "java",
+      "name": "Launch Canadian Dental Care Plan API",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "mainClass": "ca.gov.dtsstn.cdcp.api.Application",
+      "projectName": "cdcp-api",
+      "args": "",
+      "envFile": "${workspaceFolder}/.env"
```

Comment:
We don't usually use a `.env` file for Java applications.

Reply:
[Removed](https://github.com/DTS-STN/canadian-dental-care-plan/compare/e4b6b4e5d79d4f716e14946054039a3e1e8e14d9..57c82567f8066b3daa2195f0ca7ad35ecbf1b414#diff-bd5430ee7c51dc892a67b3f2829d1f5b6d223f0fd48b82322cfd45baf9f5e945)